src/pages/documentation/mobile-core/lifecycle/tabs/api-reference.md

```diff
@@ -1,7 +1,15 @@
-<Variant platform="android" api="extension-version" repeat="2"/>
+<Variant platform="android" api="extension-version" repeat="5"/>
 
 #### Java
 
+**Syntax**
+
+```java
+public static String extensionVersion()
```

Comment:
@NonNull for return type?

Reply:
The current code should have had the NonNull annotation but looks [like it was a miss](https://github.com/adobe/aepsdk-core-android/blob/main/code/lifecycle/src/phone/java/com/adobe/marketing/mobile/Lifecycle.java#L30) . I left it out here to be true to what the code.